db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/sqlgrammar.jj

```diff
@@ -4194,10 +4200,61 @@ exportStatement() throws StandardException :
 {
     StatementNode   statementNode;
     Vector parameterList = new Vector();
-}
-{
+    ValueNode exportPath;
+    ValueNode compression = null;
+    ValueNode replicationCount = null;
+    ValueNode encoding = null;
+    ValueNode fieldSeparator = null;
+    ValueNode quoteCharacter = null;
+    ValueNode quoteMode = null;
+    ValueNode fileType = null;
+}
+{
+    LOOKAHEAD ( {getToken(1).kind == EXPORT && getToken(2).kind == TO} )
```

Comment:
I think you can get rid of this, since `<EXPORT>` has two non-ambiguous branches:

1. `<EXPORT> <TO>`: the new syntax.
2. `<EXPORT> methodCallParameterList` which, by looking at `methodCallParameterList` boils down to: `<EXPORT> <LEFT_PAREN> ...`.

So it is possible to restructure it a bit differently, e.g. something like:

```
<EXPORT> handleExport
...
ValueNode handleExport() throws StandardException:
{}
{
  <TO> // new syntax
  |
  methodCallParameterList // old syntax.
}
```

(I think you're a fan of this syntax ;) )

Reply:
done :)